common/tools/armageddon/armageddon.go

```diff
@@ -823,12 +830,12 @@ func pullBlocksFromAssemblerAndCollectStatistics(userConfig *UserConfig, pullFro
 }
 
 func calculateDelayOfTx(env *common.Envelope, acceptedTime time.Time) time.Duration {
```

Comment:
why not pass the data now instead of the envelope? 
then you need to unmarshal again